### PR TITLE
Camunda Modeler v5.19.0 Release Info

### PR DESCRIPTION
**Pull request by philippfromme**
image

### CODE REVIEW

**Review by crobbins215 — COMMENTED** [review]
I read over, and it looks good from what I can see; l don't have any suggestions.

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,15 +44,16 @@ export function ReleaseInfo(props) {
...
-          <h4>Receive and Send Task Message templates</h4>
-          Message templates for Send and Receive Tasks are now supported. You can configure message name and, in the case of Receive Tasks, subscription correlation key.</li>
+          <h4>New form fields</h4>
+          Dynamic lists and tables are now available as form fields. You can use them to create forms with dynamic and static lists of data.
+        </li>
         <li>
-          <h4>More comparison operators for element templates</h4>
-          In addition to <code>equals</code> and <code>oneOf</code>, you can now use <code>isActive</code> to check if the referenced property is currently active or hidden by other conditions.
+          <h4>Simplified compensation modeling</h4>
+          Modeling of compensation handling has been made easier. You now connect a compensation boundary event to any activity that should handle compensation.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:52]:
```suggestion
          The BPMN editor ships with improved  compensation handling. Turn any activity into a compensation activity by connecting it to a  compensation boundary event.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,15 +44,16 @@ export function ReleaseInfo(props) {
...
         </li>
         <li>
-          <h4>Simplified selection outlines for connections</h4>
-          Selection outlines for single connections were removed to reduce visual noise.
+          <h4>FEEL editor improvements</h4>
+          The FEEL editor has been improved to provide better autocompletion.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:56]:
```suggestion
          The FEEL editor now offers better contextual autocompletion.
```

**Review by nikku — APPROVED** [review]
Looks great :sun_with_face:

**Pull request merged**